docs/TPM2_PCR_MEASUREMENTS.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Recognizing TPM2 PCR Measurements Made by systemd
+category: Booting
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Recognizing TPM2 PCR Measurements Made by systemd
+
+Various systemd components issue TPM2 PCR measurements during the boot process,
+both in UEFI mode and from userspace. The following lists all measurements
+done, and describes (in case done before `ExitBootServices()`) how they appear
+in the TPM2 Event Log, maintained by the PC firmware.
+
+systemd will measure to PCRs 11, 12, 13, 15.
+
+Note that the userspace measurements listed below are only done on systems
+booted with `systemd-stub`.
```

Comment:
This third paragraph should be folded into the first paragraph to form a coherent whole. Also, it's quite unclear how the measurements are done both in UEFI mode an in userspace if they're only done by `systemd-stub`.